test/exists.js

```diff
@@ -28,4 +28,13 @@ describe('#exists()', function () {
     this.fs.delete(fileDelete);
```

Comment:
This test here is testing the `exists` return false after a delete operation.

How come is it passing if you're reporting that a deletion doesn't get caught by exists?